ui/app/selectors/swap.js

```diff
@@ -196,23 +192,6 @@ export function getSwapGasButtonGroupShown (state) {
   return state.swap.gasButtonGroupShown
 }
 
-export function getSwapTitleKey (state) {
```

Comment:
**Note to reviewer:**

I think having "Swap Tokens" and "Swap ETH" is confusing. From a user's perspective, there should be no difference between the two, so I decided to set the header to just "Swap".